src/main/app/src/app/zaken/zoek/locatie-zoek/locatie-zoek.component.ts

```diff
@@ -47,6 +47,7 @@ export class LocatieZoekComponent implements OnInit, AfterViewInit, OnChanges, O
 
     private readonly DEFAULT_ZOOM: number = 8;
     private readonly MAX_ZOOM: number = 14;
+    private readonly DEFAULT_CENTER: number[] = [631711.827985, 6856275.890632];
```

Comment:
Even commentaar toevoegen hoe je aan deze getallen bent gekomen of wat ze betekenen